tests/build/build_templates_scenarios.go

```diff
@@ -129,6 +129,15 @@ func IsFBCBuild(gitUrl string) bool {
 	return false
 }
 
+func IsHermeticScenario(gitUrl string) bool {
```

Comment:
Is this function going to be used for something else once the issue is resolved?

Reply:
not sure, till the issue is fixed, we might come across situation where we have to use this function in the tests